packages/components/pie-lottie-player/README.md

```diff
@@ -0,0 +1,91 @@
+<p align="center">
```

Comment:
Please can we add some basic usage and props table info here? Basic usage in my opinion involves passing some animations to the component

Reply:
My bad, I was pretty sure to have added that.